api/src/main/resources/workbench-api.yaml

```diff
@@ -9910,25 +9910,6 @@ definitions:
         description: |-
           User's GSuite username, including appropriate domain for this environment. Uniquely describes a
           user account (but not constrained to be unique by BigQuery).
-      city:
```

Comment:
It's safe to just drop fields like this.  The corresponding fields in the BigQuery tables will continue to exist, but they will receive nulls.

I was able to test this locally because we have a `reporting_local` dataset for this purpose.  As PMI-OPS: https://console.cloud.google.com/bigquery?project=all-of-us-workbench-test&d=reporting_local&p=all-of-us-workbench-test&page=dataset&ws=!1m10!1m4!4m3!1sall-of-us-workbench-test!2sreporting_local!3slive_user!1m4!1m3!1sall-of-us-workbench-test!2sbquxjob_1260bae4_1899d70b7dc!3sUS